src/components/Gallery.js

```diff
@@ -6,7 +6,8 @@ import CustomModal from './CustomModal';
 
 const imageChangeFrequency = 15000,
       fadeOutDuration = 1000,
-      modalButtonLabel = 'Show info';
+      modalButtonLabel = 'Show info',
+      galleryToolTip = 'Click to know more!';
```

Comment:
It's important to use descriptive variable names. We should avoid magic numbers and hardcoded text too.